cnd/src/main.rs

```diff
@@ -123,8 +128,8 @@ fn main() -> Result<(), failure::Error> {
     Ok(())
 }
 
-fn derive_key_pair(secret_seed: &Seed) -> identity::Keypair {
-    let bytes = secret_seed.sha256_with_seed(&[b"NODE_ID"]);
+fn derive_key_pair(seed: &Seed) -> identity::Keypair {
+    let bytes = seed.sha256_with_seed(&[b"NODE_ID"]);
```

Comment:
❓  What is that about?

Reply:
rename _all_ instances of 'secret seed' to 'seed'.  In line with the suggestion you made on shortening the command line option from `--secret-seed-file` in `--seed-file`.

Reply:
I wondered about the `"NODE_ID"` but I see it is actually not new.

Reply:
Yes, the logic behind `sha256_with_seed()` is a mystery to me also.